solr/core/src/java/org/apache/solr/handler/component/QueryComponent.java

```diff
@@ -233,6 +233,41 @@ public void prepare(ResponseBuilder rb) throws IOException
     }
   }
 
+  private boolean allowSkipSecondGroupingStep(final GroupingSpecification groupingSpec, final boolean isReranking ) {
+    // Only possible if we only want one doc per group
+    if (groupingSpec.getGroupLimit() != 1) {
```

Comment:
minor: `getGroupLimit()` is one of the deprecated `GroupingSpecification` accessors; something like https://github.com/cpoerschke/lucene-solr/commit/5c227c41c2aa176650e19f58c351b4d538ef4fd7 could be a replacement (that commit also includes parameterised logging i.e. `{}` instead of `+`)

Reply:
Thanks, Applied in eca5af716517e